<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="liciouss-MacBook-Air-2.local" name="in.licious.dataplanning_API.DashboardWastageAPI" tests="1" failures="0" timestamp="27 Oct 2018 12:01:41 GMT" time="2.574" errors="1">
  <testcase name="dashboardwastagetest" time="2.574" classname="in.licious.dataplanning_API.DashboardWastageAPI">
    <error type="java.net.MalformedURLException" message="no protocol: 52.66.9.219:9200/_sql?sql=select sum(dispatched_quantity) from operationdataplan where to=&#039;11&#039; and source=&#039;1&#039; and createdat=&#039;2018-10-22&#039; and sku=&#039;pr_57234a4f6f77b&#039;">
      <![CDATA[java.net.MalformedURLException: no protocol: 52.66.9.219:9200/_sql?sql=select sum(dispatched_quantity) from operationdataplan where to='11' and source='1' and createdat='2018-10-22' and sku='pr_57234a4f6f77b'
	at java.net.URL.<init>(URL.java:593)
	at java.net.URL.<init>(URL.java:490)
	at java.net.URL.<init>(URL.java:439)
	at in.licious.dataplanning.ElasticSearch.readJsonFromUrl(ElasticSearch.java:28)
	at in.licious.dataplanning_API.DashboardWastageAPI.dashboardwastagetest(DashboardWastageAPI.java:48)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- dashboardwastagetest -->
</testsuite> <!-- in.licious.dataplanning_API.DashboardWastageAPI -->
